pkg/Library/Dom/container-layout.js

```diff
@@ -9,6 +9,7 @@
 import {Xen} from './Xen/xen-async.js';
 import {DragDrop} from './drag-drop.js';
 import {IconsCss} from './Material/material-icon-font/icons.css.js';
+import {deepEqual} from '../../core/js/utils/object.js';
```

Comment:
This should be from `../../core/utils.min.js`. Because this is a DOM element, it's not inside the worker. Inside the worker `arcs.min.js` is used; we made `utils.min.js` to use on the outside. 

Avoid reaching into to `core/js`, it's all bundled and minimized now. :)

I'm starting to think the `.min` files should be in `Library/Core`. This is how everything else is handled.